### ISSUE TITLE
libadwaita: leaks build date into output

### ISSUE THREAD

**Issue by raboof** [NixOS/nixpkgs#230285]
Building this package twice does not produce the bit-by-bit identical result each time, making it harder to detect CI breaches. You can read more about this at https://reproducible-builds.org/ .

Fixing bit-by-bit reproducibility also has additional advantages, such as avoiding hard-to-reproduce bugs, making content-addressed storage more effective and reducing rebuilds in such systems.

### Steps To Reproduce

```
nix-build '<nixpkgs>' -A libadwaita --check --keep-failed
```

You can use `diffoscope` to analyze the differences in the output of the two builds.

To view the build log of the build that produced the artifact in the binary cache:

```
nix-store --read-log $(nix-instantiate '<nixpkgs>' -A libadwaita)
```

### Additional context

The build date seems to be leaked into `share/metainfo/org.gnome.Adwaita1.Demo.metainfo.xml`

**Comment by Artturin** [comment]
Upstream fix https://gitlab.gnome.org/GNOME/libadwaita/-/merge_requests/881

**Comment by drupol** [comment]
This issue has been fixed in `libadwaita` 1.4.0: https://gitlab.gnome.org/GNOME/libadwaita/-/blob/1.4.0/demo/data/meson.build?ref_type=tags

**Comment by oliviacrain** [comment]
[`libadwaita` was upgraded to 1.4.0 last month.](https://github.com/NixOS/nixpkgs/commit/b1bb52acf523045722433fdea189525fb622c934)

As expected, I can't reproduce this failure. We can close this.
